init.sh

```diff
@@ -12,16 +12,17 @@ TRACE=""
 # validate dependencies are installed
 command -v jq > /dev/null 2>&1 || { echo >&2 "jq not installed. More info: https://stedolan.github.io/jq/download/"; exit 1; }
 
-# Reinstall daemon
+# remove existing daemon
 rm -rf ~/.evmosd*
```

Comment:
maybe we shouldn't remove the daemon

Reply:
still drafting this PR. The init.sh changes were in there by accident. You mean we shouldn't reinstall it?

Reply:
yeah that's also ok 👍